### PR TITLE
Updated with Company report sample example

### PR DESCRIPTION
**Pull request by venkattejagaddam**
Updated with Company report sample example and edited main file by removing unnecessary comments.

### CODE REVIEW

File: TypescriptExample/companyReport.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+"use strict";
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```
Comment (tspence) [inline TypescriptExample/companyReport.js]: This looks like the compiled typescript file.  Should we commit this?

Comment (venkattejagaddam) [reply TypescriptExample/companyReport.js]: I will just make another pull request with only typescript files.

Comment (venkattejagaddam) [reply TypescriptExample/companyReport.js]: I had deleted the compiled typescript files.

**Comment by tspence** [comment]
Looks good except for the conflicts.  Could you fix those so I can merge?

**Pull request closed without merging**